Containerfile.toolbox

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/toolbx-images/ubuntu-toolbox:22.04
+# From https://github.com/toolbx-images/images/tree/main/ubuntu/22.04
+
+LABEL com.github.containers.toolbox="true" \
+      usage="This image is meant to be used with the toolbox or distrobox command" \
+      summary="A cloud-native terminal experience" \
+      maintainer="jorge.castro@gmail.com"
+
+RUN sed -Ei '/apt-get (update|upgrade)/s/^/#/' /usr/local/sbin/unminimize && \
+    apt-get update && \
+    yes | /usr/local/sbin/unminimize && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install \
+        $(cat extra-packages | xargs) && \
+    rm -rd /var/lib/apt/lists/*
+
+RUN rm /toolbox-packages
+
+RUN   ln -fs /bin/sh /usr/bin/sh && \
```

Comment:
Will adding `ln -fs /bin/sh /usr/bin/sh` mean that any shell scripts containing `#!/usr/bin/env sh` will be ran on the host instead of the container?